### PR TITLE
Fix missing uninstall calls for tooltips

### LINKED ISSUE #1421: Potential Tooltip leak
`Tooltip.install` gets called in some places without corresponding `Tooltip.uninstall`.

### PR DESCRIPTION
**Pull request by Katharsas**
Fixes #1421

### CODE REVIEW

**Comment by 1-alex98** [comment]
Lets see if that was it but I can imagine so. There where lots of tooltips in memory dump

File: src/main/java/com/faforever/client/game/GamesTilesContainerController.java [hunk 1]
```diff
@@ -68,7 +67,11 @@ public GamesTilesContainerController(UiService uiService, PreferencesService pre
       JavaFxUtil.assertApplicationThread();
         while (change.next()) {
           change.getRemoved().forEach(gameInfoBean -> {
-            boolean remove = tiledFlowPane.getChildren().remove(uidToGameCard.remove(gameInfoBean.getId()));
+            Node card = uidToGameCard.remove(gameInfoBean.getId());
+            if (card != null) {
+              Tooltip.uninstall(card, tooltip);
+            }
+            boolean remove = tiledFlowPane.getChildren().remove(card);
```
Comment (Brutus5000) [inline src/main/java/com/faforever/client/game/GamesTilesContainerController.java]: The following three lines should go into the card!=null block
The else block should also throw an error in the log.
(We don't know how many times it actually occurs and that would help I guess)

**Review by Brutus5000 — CHANGES_REQUESTED** [review]

**Review by Brutus5000 — APPROVED** [review]

**Review by 1-alex98 — APPROVED** [review]

**Comment by 1-alex98** [comment]
@Katharsas did you check if tooltips are removed in chat as well?

**Comment by 1-alex98** [comment]
I think the tooltips are never removed but i think the difference is that the tooltip is not held by an external Controller

**Comment by 1-alex98** [comment]
So should be fine

**Comment by Katharsas** [comment]
We basically made a search for `*.install` in all sources and made sure that every install has corresponding uninstall. Don't remember if we found chat stuff.

**Pull request merged**